api/horizon.md

```diff
@@ -21,7 +21,7 @@ const hz = Horizon();
 All arguments are optional. Pass them to `Horizon` in an object with option keys: `{secure: true}`.
 
 * `host`: the hostname of the Horizon server. This defaults to `window.location`, i.e., the machine that served the application to the client.
-* `secure`: a boolean indicating whether the server should use secure websockets. Defaults to `true`.
+* `secure`: a boolean indicating whether the server should use secure websockets. Defaults to `false` unless origin is using HTTPS.
```

Comment:
Maybe we should say "The default is `true` if `window.location` uses `https` and `false` otherwise."?
@chipotle what do you think?